proto/api.proto

```diff
@@ -1828,6 +1850,41 @@ message GetJupiterPricesResponse {
   repeated TokenPriceV2 tokenPrices = 1;
 }
 
+message GetZetaTransactionStreamRequest {
+  repeated string instructions = 1;
+}
+
+message Transaction {
```

Comment:
Do we need to add more "Zeta" suffixes to these messages to identify them as belonging to the Zeta domain (as opposed to, say, Raydium)?